src/algebra/lie/basic.lean

```diff
@@ -836,6 +842,30 @@ instance : add_comm_monoid (lie_submodule R L M) :=
 
 @[simp] lemma add_eq_sup (N N' : lie_submodule R L M) : N + N' = N ⊔ N' := rfl
 
+section inclusion_maps
+
+/-- The inclusion of a Lie submodule into its ambient space is a morphism of Lie modules. -/
+def incl : N →ₗ⁅R,L⁆ M :=
+{ map_lie := λ x m, rfl,
+  ..submodule.subtype (N : submodule R M) }
+
+@[simp] lemma incl_apply (m : N) : N.incl m = m := rfl
+
+lemma incl_eq_val : (N.incl : N → M) = subtype.val := rfl
+
+variables {N} {N' : lie_submodule R L M} (h : N ≤ N')
+
+/-- Given two nested Lie submodules `N ⊆ N'`, the inclusion `N ↪ N'` is a morphism of Lie modules.-/
+def of_le : N →ₗ⁅R,L⁆ N' :=
```

Comment:
I think that in other places of the library we call this `hom_of_le`. That seems slightly more readable (but also longer, of course).

Reply:
Thanks, I prefer `hom_of_le` and indeed I see it used in the category theory part of the library.